### ISSUE TITLE
small object detect use the file yolov4-tiny-3l.cfg but one yolo layer‘s IOU is 0.00000

### ISSUE THREAD

**Issue by tigerdhl** [AlexeyAB/darknet#7352]
everying is ok when I train my dataset used the yolov4-tiny-custom.cfg, but the small object detect is no so good. So I tried the yolov4-tiny-3l.cfg file to improve the effect. And the train information seem some problem with the first yolo layer's IOU output is 0.000000.
It appeared  at the training start, I don't understand, does it has influence in the last result?

v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 30 Avg (IOU: 0.000000), count: 1, class_loss = 0.000006, iou_loss = 0.000000, total_loss = 0.000006
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 37 Avg (IOU: 0.586642), count: 4, class_loss = 0.625391, iou_loss = 0.084107, total_loss = 0.709498
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 44 Avg (IOU: 0.532751), count: 6, class_loss = 1.048514, iou_loss = 3.818012, total_loss = 4.866526
 total_bbox = 62765, rewritten_bbox = 0.035051 %
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 30 Avg (IOU: 0.000000), count: 1, class_loss = 0.000005, iou_loss = 0.000000, total_loss = 0.000005
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 37 Avg (IOU: 0.468625), count: 3, class_loss = 0.649765, iou_loss = 0.051397, total_loss = 0.701163
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 44 Avg (IOU: 0.572519), count: 8, class_loss = 1.683683, iou_loss = 2.950364, total_loss = 4.634048

**Comment by tigerdhl** [comment]
I get the question, the "num" in the yolo layer of .cfg file should be 9.
